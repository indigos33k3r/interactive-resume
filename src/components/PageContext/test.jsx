import React from 'react'
import {shallow} from 'enzyme'

// TODO:
// Add tests for #componentDidUpdate once Enzyme is update to support `contextType`
// https://github.com/airbnb/enzyme/issues/1913
describe('PageContext', () => {
  it('renders <PageContext />', () => {
    const component = render()

    expect(component).toMatchSnapshot()
  })

  function render() {
    const PageContext = require('.').default

    return shallow(<PageContext />)
  }
})
