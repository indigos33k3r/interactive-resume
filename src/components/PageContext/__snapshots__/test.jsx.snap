// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageContext renders <PageContext /> 1`] = `
<div
  className="page-context"
  style={null}
>
  <Player />
  <Background>
    <Building
      buildingClass="tunnel"
    />
    <Lamps />
    <Trees />
    <Building
      buildingClass="welcome"
    >
      <Person
        personClass="beach1"
      />
      <Person
        personClass="beach2"
      />
      <Person
        personClass="guy1-briefcase"
      />
      <Person
        personClass="guy2-bag"
      />
    </Building>
    <Building
      buildingClass="newsstand"
    >
      <Person
        personClass="guy3-briefcase"
      />
      <Person
        personClass="guy4-newspaper"
      >
        <PageContextNewspaper />
      </Person>
    </Building>
    <Building
      buildingClass="xp"
    >
      <Person
        personClass="guy5-jumping1"
      >
        <JumpingShadow />
      </Person>
      <Person
        personClass="guy6-jumping2"
      >
        <JumpingShadow />
      </Person>
      <Person
        personClass="guy7-cheering"
      >
        <PageContextXp />
      </Person>
      <Person
        personClass="girl8-sitting"
      />
      <Person
        personClass="guy9-fishing"
      />
    </Building>
    <Building
      buildingClass="webdev"
    >
      <Person
        personClass="guy10-bag"
      />
      <Person
        personClass="guy11-bag"
      />
      <Person
        personClass="girl12-laptop"
      >
        <PageContextWebdev />
      </Person>
      <Person
        personClass="guy13-jumping"
      >
        <JumpingShadow />
      </Person>
    </Building>
    <Building
      buildingClass="arcade"
    >
      <Person
        personClass="guy14-skating"
      />
      <Person
        personClass="girl15-gameboy"
      >
        <PageContextGames />
      </Person>
    </Building>
    <Building
      buildingClass="mobile"
    >
      <Person
        personClass="guy16-manhole"
      />
      <Person
        personClass="guy17-photo"
      >
        <PageContextMobile />
      </Person>
      <Person
        personClass="radio"
      />
      <Person
        personClass="pair18-photo"
      />
    </Building>
    <Building
      buildingClass="lab"
    >
      <Person
        personClass="guy19-icecream"
      />
      <Person
        personClass="guy20-mib"
      />
      <Person
        personClass="guy21-doc"
      >
        <PageContextLabel />
      </Person>
      <Person
        personClass="guy22-doc2"
      />
    </Building>
    <Building
      buildingClass="contact"
    >
      <PageContextContact />
    </Building>
    <Building
      buildingClass="police-line"
    >
      <Vehicle
        vehicleClass="police-x"
      />
      <Vehicle
        vehicleClass="police-y"
      />
    </Building>
  </Background>
</div>
`;
